packages/wrapper/src/vscode/services.ts

```diff
@@ -50,7 +47,20 @@ export const configureServices = async (config: VscodeServicesConfig): Promise<I
             }
         };
     }
-    mergeServices(config.specificServices ?? {}, serviceConfig.userServices);
+    mergeServices(config.specificServices, serviceConfig.userServices);
+
+    // set the log-level via the development settings
+    const devLogLevel = serviceConfig.workspaceConfig.developmentOptions?.logLevel;
+    if (serviceConfig.workspaceConfig.developmentOptions?.logLevel === undefined) {
```

Comment:
you probably mean `devLogLevel` instead of `serviceConfig.workspaceConfig.developmentOptions?.logLevel`

Reply:
oh, yes, I forgot to replace that 😅

Reply:
Done ⬇️